### PR TITLE
Added songs list api

### PR DESCRIPTION
**Pull request by girdhariag**

### CODE REVIEW

File: lib/getsongslist.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  request(options, (err, response, body) => {
+    if (err) throw new Error(err);
+
+    if (body) {
+      const songs = JSON.parse(body);
+      const { data } = songs;
+      const [first] = data;
+      const song = {
+        name: first.name,
+        thumbnail: first.pictures.thumbnail,
```
Comment (astronomersiva) [inline lib/getsongslist.js]: `pictures.thumbnail` looks too small on desktop Messenger. Can we settle for `pictures.medium_mobile`?

File: lib/getsongslist.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+const request = require('request');
+
+module.exports = function getSongsList(keyArray) {
+  const keysJoined = keyArray.join('+');
```
Comment (astronomersiva) [inline lib/getsongslist.js]: @girdhariag , there is a chance of the keyArray being something like `['sicilian pizza', 'pepperoni']`(returned by Google's Vision API). When you do a `keyArray.join( )`, keysJoined will be `sicilian%20pizza+pepperoni` which doesn't work with our songs API. A quick workaround would be doing something like `keyArray.join('+').replace(' ', '+')`. If you have a better solution for this, that would be awesome as well.

I will merge this PR once this is fixed 👍

File: lib/getsongslist.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+      const songs = JSON.parse(body);
+      const { data } = songs;
+      const [first] = data;
+      const song = {
+        name: first.name,
+        thumbnail: first.pictures.thumbnail,
+        url: first.url,
+      };
+
+      return song;
```
Comment (astronomersiva) [inline lib/getsongslist.js]: Can you change it to return a promise instead? 

`resolve` the song or the empty object below and `reject` in case of err.

**Review by astronomersiva — CHANGES_REQUESTED** [review]

File: lib/getsongslist.js (hunk 1 above)
Comment (girdhariag) [reply lib/getsongslist.js]: I am keeping the property key as thumbnail only.

Comment (astronomersiva) [reply lib/getsongslist.js]: Yeah, no problem with that.

**Review by astronomersiva — APPROVED** [review]
:1st_place_medal:

**Pull request merged**